internal/svcmanager/svcmanager.go

```diff
@@ -101,5 +115,10 @@ func (m *Manager) ping(ctx context.Context) error {
 	if err := client.RunWithContext(ctx, r, &resp); err != nil {
 		return err
 	}
+
+	if m.checkVersion && resp.Version.State.Version != constants.Version && resp.Version.State.Branch != constants.BranchName {
+		return &errVersionMismatch{locale.NewError("err_ping_version_mismatch")}
```

Comment:
Nit: Just occurred to me, you don't really need the error type. Since you're not wrapping another error or need any custom methods could use: 

```
var MyError = locale.NewError("err_myerror")
errors.Is(err, MyError)
```

That should work even if `MyError` ends up getting wrapped itself.